courses/free-courses-en.md

```diff
@@ -707,6 +708,7 @@
 ### Kubernetes
 
 * [Kubernetes Core Concepts](https://kube.academy/paths/kubernetes-core-concepts) - KubeAcademy (VMware)
+* [Kubernetes Course](https://www.youtube.com/watch?v=d6WC5n9G_sM) - Bogdan Stashchuk (YouTube)
```

Comment:
- Use titles as it is...
- Complete creators
- YT playlist / videos don't need resource notes

```suggestion
* [Kubernetes Course - Full Beginners Tutorial (Containerize Your Apps!)](https://www.youtube.com/watch?v=d6WC5n9G_sM) - 
freeCodeCamp, `tch.:` Bogdan Stashchuk (YouTube)
```

Reply:
```suggestion
* [Kubernetes Course](https://www.youtube.com/watch?v=d6WC5n9G_sM) - Bogdan Stashchuk (FreeCoodeCamp)
```